internal/core/saptune/saptune_test.go

```diff
@@ -0,0 +1,80 @@
+package saptune
+
+import (
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/suite"
+	"github.com/trento-project/agent/pkg/utils/mocks"
+)
+
+type SaptuneTestSuite struct {
+	suite.Suite
+}
+
+func TestSaptuneTestSuite(t *testing.T) {
+	suite.Run(t, new(SaptuneTestSuite))
+}
+
+func (suite *SaptuneTestSuite) TestNewSaptune() {
```

Comment:
I'm a bit confused with the tests.
I would like to have some tests for the `NewSaptune` and other tests for the `RunCommand`.
They don't need to be combined. For the `RunCommand` you can create the saptune struct of your convinience